packages/core/src/components/cv-inline-notification/cv-inline-notification-notes.md

```diff
@@ -10,7 +10,7 @@ http://www.carbondesignsystem.com/components/notification/code
 <cv-inline-notification
   kind="error"
   title="notification title"
-  sub-title="a subtitle"
+  sub-title="a <a>subtitle</a>"
```

Comment:
Add `href="#"` to examples so the mouse cursor looks correct.

Reply:
```suggestion
  sub-title="a <a href="#">subtitle</a>"
```